ldms/src/ldmsd/ldmsd_prdcr.c

```diff
@@ -845,6 +845,17 @@ static void prdcr_connect(ldmsd_prdcr_t prdcr)
 {
 	int ret;
 
+	if (prdcr_resolve(prdcr->host_name, prdcr->port_no, &prdcr->ss, &prdcr->ss_len)) {
+		if (0 == prdcr->not_resolved_cnt) {
+			ldmsd_log(LDMSD_LERROR, "Producer '%s' connection failed. Unable to resolve hostname '%s:%u'.\n",
+				prdcr->obj.name, prdcr->host_name,(unsigned) prdcr->port_no);
+		}
+		prdcr->not_resolved_cnt++;
```

Comment:
@nichamon, We should talk about this. I don't agree that we should only report this once. The service comes and goes and we only have one user who complained about the possibility of their configuration error being reported over and over. Let's consult with other users in the UG meeting on Monday.